.github/workflows/deploy-dev.yml

```diff
@@ -1,6 +1,8 @@
 name: Deploy to Dev (ECS)
 
 on:
+  schedule:
+    - cron: '0 5 * * *'
   workflow_dispatch:
```

Comment:
Could you please switch this to `dev-blue` to leave dev open for staging purposes?

Reply:
@bklebe All set!